setup-scripts/log-checker.py

```diff
@@ -40,13 +41,27 @@ async def on_ready():
 async def run_checks():
     print("Running Skynet portal log checks")
     try:
+        await check_load_average()
         await check_docker_logs()
 
     except: # catch all exceptions
         trace = traceback.format_exc()
         await send_msg(client, "```\n{}\n```".format(trace), force_notify=False)
 
 
+# check_load_average monitors the system's load average value and issues a
+# warning message if it exceeds 10.
+async def check_load_average():
+    uptime_string = os.popen("uptime").read().strip()
+    # pattern = ""
+    if sys.platform == "Darwin":
+        pattern = "^.*load averages: \d*\.\d* \d*\.\d* (\d*\.\d*)$"
+    else:
```

Comment:
Are all other OS the same syntax, and it is really just Darwin that is different? 
If this is trying to distinguish between just Mac and Linux I would change this to specifically check those two OS and error otherwise.

Reply:
I *think* all Linux distros use the same format. They basically share the same code. But Macs are not Linux, they are FreeBSD (or they were), so they use a different implementation.